src/EnergyPlus/EvaporativeFluidCoolers.cc

```diff
@@ -1133,19 +1123,14 @@ namespace EvaporativeFluidCoolers {
 
     void EvapFluidCoolerSpecs::getDesignCapacities(EnergyPlusData &state, const PlantLocation &, Real64 &MaxLoad, Real64 &MinLoad, Real64 &OptLoad)
     {
-        if (this->Type == DataPlant::PlantEquipmentType::EvapFluidCooler_SingleSpd) {
-
-            MinLoad = 0.0; // signifies non-load based model (i.e. forward)
-            MaxLoad = this->HighSpeedStandardDesignCapacity * this->HeatRejectCapNomCapSizingRatio;
-            OptLoad = this->HighSpeedStandardDesignCapacity;
-
-        } else if (this->Type == DataPlant::PlantEquipmentType::EvapFluidCooler_TwoSpd) {
-
+        switch (this->Type) {
```

Comment:
The IF and ELSEIF sections were actually identical, this is nice and clean now.

Reply:
At this point just use `if-else`